ts/components/InstabugButtons.tsx

```diff
@@ -75,7 +96,7 @@ const mapStateToProps = (state: GlobalState) => ({
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   dispatchIBReportOpen: (type: string) =>
```

Comment:
```suggestion
  dispatchIBReportOpen: (type: InstabugReportType) =>
```

Reply:
Why this? I left the string type and pass "bug" or "chat"...